base_station/gui/src/components/ToggleButton.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+    <label :for="id + '_button'" :class="{'active': isActive}" class="toggle__button">
+        <span v-if="isActive" class="toggle__label">{{ enableText }}</span>
+        <span v-if="! isActive" class="toggle__label">{{ disabledText }}</span>
+
+        <input type="checkbox" :disabled="disabled" :id="id + '_button'" v-model="checkedValue">
+        <span class="toggle__switch"></span>
+    </label>
+</template>
+
+<script>
+    export default {
+    props: {
+        disabled: {
+            type: Boolean,
+            default: false
+        },
+        labelEnableText: {
```

Comment:
Can you make the endabled and disabled text props required instead of having defaults?